main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -140,6 +140,7 @@ class Flix {
     "Chain.flix" -> LocalResource.get("/src/library/Chain.flix"),
     "Char.flix" -> LocalResource.get("/src/library/Char.flix"),
     "Choice.flix" -> LocalResource.get("/src/library/Choice.flix"),
+    "Closeable.flix" -> LocalResource.get("/src/library/Closeable.flix"),
```

Comment:
Since this will be a sugary feature, we might need to add it to the `MinLib` too.

Reply:
Good point.

Reply:
Hmm. Actually, I'm not so sure about that now I think about it. It's only used if something `Closeable` is ever created, and I don't imagine that anything `Closeable` will ever be in `MinLib`?

Reply:
I'm going to mark this as resolved: we can add it to MinLib as/when it becomes necessary.

Reply:
Agreed.